pep/app/services/ajax.ts

```diff
@@ -19,14 +20,16 @@ export default class AjaxService extends Service {
      * Add the oauth token authorization header to all requests
      * @returns {Object}
      */
-    @computed('session.{isAuthenticated,data.authenticated.access_token}')
+    @computed('session.{isAuthenticated,data.authenticated.SessionId}')
     get authorizationHeaders() {
         const headers = {} as any;
         // api auth token is sent in cookies
-        // if (this.session.isAuthenticated) {
-        //     const { access_token } = this.session.data!.authenticated;
-        //     headers['Authorization'] = `Bearer ${access_token}`;
-        // }
+        if (this.session.isAuthenticated) {
+            // We are converting to unknown because session data is specified as something
+            // completely different by the addon
+            const { SessionId } = (this.session.data!.authenticated as unknown) as PepSecureAuthenticatedData;
```

Comment:
same here as the last comment, if we just adjust our already custom session data interface, we won't need to convert it everywhere.